### ISSUE TITLE
refactor(orc8r): remove unnecessary reindexer functionality

### ISSUE THREAD

**Issue by reginawang3495** [magma/magma#10293]
- remove `RunUnsafe`
- support non-nil `sendUpdate` from Run
- use `sendUpdate` in tests instead of `TestHookReindexSuccess`

See `TODO(#10293)` for code pointers

**Comment by reginawang3495** [comment]
blocked on #10294

**Comment by shahrahman-fb** [comment]
Closing this issue as part of a repository cleanup effort. This issue has had no activity for over 2 years.

If this issue is still relevant, please reopen it or file a new issue with updated information. We are actively maintaining the project again and want to focus on current, actionable issues.

See: #15833 (Phase 1 cleanup), #15828 (tracking issue)
